src/mocks/handlers/stac.ts

```diff
@@ -0,0 +1,11 @@
+import { HttpResponse, http } from 'msw';
+
+import { getCollections } from '../fixtures/stac';
+
+const API_URL = '/api/catalogue/stac/collections';
+
+const getStacCollectionsData = http.get(`*${API_URL}*`, () => HttpResponse.json(getCollections()));
+
+const handlers = [getStacCollectionsData];
+
+export default handlers;
```

Comment:
I've seen you mention before you don't like the default exports, would you rather `export const handlers = [getStacCollectionsData]`

Reply:
In this situation, it is the preferred way by Mock Service Worker, so I just stuck with it. I don't know if it is like NextJS in that Pages must be `default exports`, but it is definitely what the docs have and also what others have done, when I've looked at stackoverflow questions etc. My decision in this particular one was to go with the masses, rather than me going off on my own, which is why it is like this.